types/session_cost.yaml

```diff
@@ -1,5 +1,38 @@
 description: Types to provide the session price
 types:
+  IdlePrice:
+    description: >-
+      Price components when idle
+    type: object
+    additionalProperties: false
+    properties:
+      graceMinutes:
+        description: >-
+          Grace period in minutes before idle time is charged.
+          Grace minutes start counting from the timestamp of the message in which state
+          changed from "Charging" to "Idle".
+        type: integer
+      hourPrice:
+        description: >-
+          Price per hour while idle
+        type: number
+  ChargingPriceComponent:
```

Comment:
I kept it but changed the enum type to 'category'. Because we need an array with this information in SessionCost (otherwise I have to define it there, which is also fine by me).

Reply:
A SessionCostChunk is the 'real cost' of the session (the total cost of the charging session until now for example), ChargingPriceComponent is the price per kWh, hour or flat fee. So if you want to calculate the total cost (for example if the charging station goes offline), you need the charging price components to be able to do that.

Reply:
understood - but can it be modeled similarly to the SessionCostChunk and also named similarly, e.x. `PriceChunk` and allow the user to pass an array of chunks

Reply:
Well that is only renaming ChargingPriceComponent to PriceChunk isn't it? As it already is an array.